src/main/java/usdot/fhwa/stol/c2c/c2c_mvt/controllers/StandardValidationController.java

```diff
@@ -58,6 +58,7 @@
 import usdot.fhwa.stol.c2c.c2c_mvt.C2CMVTApplication;
 import usdot.fhwa.stol.c2c.c2c_mvt.decoders.Decoder;
```

Comment:
It is  a bit confusing to have both `import com.github.erosb.jsonsKema.JsonParser` and self defined `JsonParser.java`

Reply:
The methods that end up using our JsonParser are abstracted to only use the base class Parse,r so there isn't any name collision in this file. I guess we could rename the parser and decoders by prepending C2CMVT but I think that is unnecessary.